src/geff/valid_values.py

```diff
@@ -95,7 +98,7 @@ def validate_space_unit(unit_name: str) -> bool:
         bool: True if a space unit is a KNOWN valid unit.
         False if the unit is not known. The unit may be valid.
     """
-    return unit_name in VALID_SPACE_UNITS
+    return unit_name in VALID_SPACE_UNITS or ureg(unit_name).check("[length]")
```

Comment:
Can you expand this docstring (and the one for time) to mention checking against pint?

Reply:
I expanded the doc string in https://github.com/live-image-tracking-tools/geff/pull/285/commits/8452df1670c8e57860a1ded32c5324e73baee238